api/src/main/java/org/openmrs/module/csaudecore/interceptor/entityhandler/util/ObservationUtils.java

```diff
@@ -0,0 +1,156 @@
+package org.openmrs.module.csaudecore.interceptor.entityhandler.util;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Objects;
+import java.util.UUID;
+
+import org.openmrs.CodedOrFreeText;
+import org.openmrs.Concept;
+import org.openmrs.Condition;
+import org.openmrs.Encounter;
+import org.openmrs.Obs;
+import org.openmrs.Patient;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.csaudecore.util.CSaudeCoreConstants;
+import org.openmrs.module.csaudecore.util.EncounterUtils;
+
+public class ObservationUtils {
```

Comment:
Podia-se tornar esta classe num `@Component` com métodos de instância em vez de métodos estáticos.
```suggestion
@Component
public class ObservationUtils {
```

Reply:
@ynurmahomed  feito